src/main/java/org/springframework/data/repository/query/Parameter.java

```diff
@@ -51,6 +54,21 @@
 	private final boolean isDynamicProjectionParameter;
 	private final Lazy<Optional<String>> name;
 
+	static {
+
+		List<Class<?>> types = new ArrayList<>(Arrays.asList(Pageable.class, Sort.class));
+
+		try {
+			// consider Kotlin Coroutines Continuation a special parameter. That parameter is synthetic and should not get
+			// bound to any query.
+			types.add(ClassUtils.forName("kotlin.coroutines.Continuation", Parameter.class.getClassLoader()));
```

Comment:
Shall we extract this into a reusable `o.s.data.util.ReflectionUtils.ifPresent(String classname, Consumer<Class<?>> consumer)`?